telegram/contact.py

```diff
@@ -39,7 +39,7 @@ class Contact(TelegramObject):
 
     """
 
-    def __init__(self, phone_number, first_name, last_name='', user_id=0, **kwargs):
+    def __init__(self, phone_number, first_name, last_name=None, user_id=0, **kwargs):
```

Comment:
I think `user_id` should be default to `None`, same reasons like with `Chat` above.